src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/meta/created.js

```diff
@@ -0,0 +1,22 @@
+ 'use strict';
+/**
+ * Displays the created at meta information
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+define(
+    [
+        'underscore',
+        'pim/form/common/meta/created',
+        'text!pim/template/product/meta/created'
+    ],
+    function (_, Created, template) {
+        return Created.extend({
```

Comment:
To have specific class name and specific template.
In the future, there will be no more parent class, only this one.